rspace/src/main/scala/coop/rchain/rspace/IStore.scala

```diff
@@ -69,6 +69,19 @@ trait IStore[C, P, A, K] {
 
   val trieStore: ITrieStore[T, Blake2b256Hash, GNAT[C, P, A, K]]
 
+  private[rspace] def pruneHistory(in: Seq[TrieUpdate[C, P, A, K]]): Seq[TrieUpdate[C, P, A, K]] =
```

Comment:
Nice and simple!

Reply:
I'm debating with myself over whether or not this function should be in `IStore`.  Thoughts?

Reply:
Private in `package object history`? Name could be shortened to `prune`